chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -121,6 +121,9 @@ errorReport.repoCannotBeContacted	= Repository cannot be contacted
 
 
 attachmentBasePath.selectDir                                   = Choose Base Directory
+attachmentBasePath.libraryAttachmentBasePaths.label            = Library Attachment Base Directories
+attachmentBasePath.libraryAttachmentBasePaths.relative         = Relative
+attachmentBasePath.libraryAttachmentBasePaths.path             = Path
```

Comment:
I must be missing something with the localization since I got an error when I tried to reference these in the xul.